templates/articles/quality_management.html

```diff
@@ -0,0 +1,63 @@
+
+<p>The chart presented below underscores disparities between project quality management in agile and predictive methodologies. It commences by reiterating the distinctions between predictive and agile approaches.</p>
+
+<figure class="figure">
+	<img class="figure-img"
+		 src="https://projectleadershipwaterloo.files.wordpress.com/2019/12/predictive-vs-agile-quality.png?w=640&h=635"
+		 alt="table" width="500" height="500">
+
+	<figcaption class="figure-caption">
+		Sourced from <a href="https://projectleadershipwaterloo.files.wordpress.com/2019/12/predictive-vs-agile-quality.png?w=640&h=635">https://projectleadershipwaterloo.files.wordpress.com/2019/12/predictive-vs-agile-quality.png?w=640&h=635</a>
+	</figcaption>
+</figure>
+
+<h1>Acceptance Criteria</h1>
+
+<p>User stories serve as a valuable instrument for comprehending the client or customer's needs. When acceptance criteria are incorporated into these user stories, they offer additional clarity for the user requirements. Acceptance Criteria should be constructed in a manner that is testable. These requirements help form what needs to be done for the project management team. These tests should be clear-cut determinations of success or failure, leaving no room for interpretation. It’s very important that everyone is able to comprehend your acceptance criteria. If the team is unable to grasp it, the criteria becomes ineffective. If there’s doubts about the clarity of a given criteria, time should be invested to inquire and make adjustments until proper clarity is achieved. Acceptance criteria should offer a user’s perspective. They serve to look at the problem in question through the lens of a customer. They ought to be crafted within the context of an actual user’s perspective.</p>
+
+<h1>The Role of the Product Owner</h1>
+
+<p>Within an agile framework, the product owner assumes the responsibility of prioritizing and supervising the tasks of the project team. The overarching goal here is to ensure that the company maximizes the value derived from the effort of the project team. Remember that an agile team also shoulders responsibilities for the project, meaning that any blowback will reflect poorly on the entire team, rather than scapegoating the product owner.</p>
+
+<figure class="figure">
+	<img class="figure-img"
+		 src="https://www.productplan.com/uploads/The-Agile-Product-Owner-WIP.png"
+		 alt="Product Owner" width="500" height="500">
+
+	<figcaption class="figure-caption">
+		Sourced from <a href="https://www.productplan.com/uploads/The-Agile-Product-Owner-WIP.png">https://www.productplan.com/uploads/The-Agile-Product-Owner-WIP.png</a>
+	</figcaption>
+</figure>
+
+<p>Lets delve into specifics of the responsibilities of a product owner: </p>
+
+<ol type="1">
```

Comment:
Nit: It looks like `1` is the default `ol` `type`, so I think you can omit this.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol#type